RELEASE_NOTES.txt

```diff
@@ -2,6 +2,238 @@ Apache Ignite Release Notes
+* Fixed failed WalModeChangeAdvancedSelfTest.testMaintenanceIsSkippedIfWasFixedManuallyOnDowntime
+* Fixed few issues of command warm-up in control.sh
+* Fixed handling of a rebalancing with disabled WAL
+* Fixed help information for control.sh --property help
+* Fixed high contention on GridCachePartitionExchangeManager.ExchangeFutureSet#values.
+* Fixed historical iterator for atomic group should transfer few more rows than required
+* Fixed incorrect calculation of WAL segments that should be deleted from WAL archive
+* Fixed incorrect topology snapshot logger output about coordinator change
+* Fixed incorrect usage of Class.isAssignableFrom in SystemViewLocal and SystemViewMBean classes
+* Fixed incorrect work of predicates (< and >) in where clause with compound primary key
+* Fixed local metastorage system view failure if unmarshalled values present
+* Fixed metastore which may leave uncompleted write futures during node stop
+* Fixed model classes which require manual deserialization if used inside Job loaded by p2p
+* Fixed operations started on client nodes are not traced
+* Fixed partial index rebuild fails in case indexed cache contains different data types
+* Fixed partition loss detection is not working if a dynamic cache is not started on a client
+* Fixed performance drop when there are many thin clients per server
+* Fixed pingNode(InetSocketAddress, UUID, UUID) failure pinging nodes with unresolved addresses
+* Fixed print warning if unordered map used for bulk update operation on atomic cache
+* Fixed rebalance might hangs in no final state though all partitions are owned
+* Fixed release a segment if it is not present in FileWriteAheadLogManager#reserve
+* Fixed removing expired cache entries
+* Fixed restorePartitionStates triggers FailureHandler on node startup
+* Fixed rows buffers usage by reducing index for simple one-way reducer
+* Fixed scan query failure with an assertion error: Unexpected row key
+* Fixed security issue in JMX configuration using ignite.sh
+* Fixed speed base throttling unexpectedly degraded to zero
+* Fixed spring data integration which doesn't introspect the fields of the key object
+* Fixed starvation in mgmt pool caused by  MetadataTask execution
+* Fixed synchronization issues when different classloaders are used for deployment of same class
+* Fixed transaction commit completes successfully if partition is lost on commit phase
+* Fixed validate index does not stop after process of control.sh was interrupted
+* Fixed validateKeyAndValue which attempts to deserialize key and value when QueryEntity.fields is not set
+* Fixed waiting for completion of operations on indexes before cache stop
+* Moved ClusterMetrics to the new metrics framework
+* Moved TcpDiscoveryStatistics to new metrics framework
+* Moved spring-data modules to ignite-extensions
+* Removed 'default' cache usage from REST API
+* Removed WALPointer and use FileWALPointer
```

Comment:
Not sure if I get it right. What does "use" imply here?

Reply:
Fixed.